packages/scripts/deploy-api.sh

```diff
@@ -19,58 +19,12 @@ if [[ -z "${ECS_SERVICE}" ]]; then
   exit 1
 fi
 
-# Fail on error
-set -e
-
-# Echo commands
-set -x
-
-aws ecr get-login-password --region $AWS_REGION | docker login --username AWS --password-stdin $ECR_REPO_URI
-
-GITHUB_SHA=$(git rev-parse --short HEAD)
-
-API_FOLDER=packages/api
-TARBALL_NAME=metriport-api.tar.gz
-API_TARBALL=$API_FOLDER/$TARBALL_NAME
-
-rm -rf ${API_TARBALL}
-
-# Build server tarball
-tar \
-  --exclude='**/*.ts' \
-  --exclude='**/__tests__' \
-  --exclude='**/*.spec*' \
-  --exclude='**/*.test*' \
-  --exclude="**/${TARBALL_NAME}" \
-  -czf ${API_TARBALL} \
-  package.json \
-  package-lock.json \
-  packages/shared/package.json \
-  packages/shared/dist \
-  packages/core/package.json \
-  packages/core/dist \
-  packages/api-sdk/package.json \
-  packages/api-sdk/dist \
-  packages/commonwell-sdk/package.json \
-  packages/commonwell-sdk/dist \
-  packages/carequality-sdk/package.json \
-  packages/carequality-sdk/dist \
-  packages/ihe-gateway-sdk/package.json \
-  packages/ihe-gateway-sdk/dist \
-  ${API_FOLDER}/package.json \
-  ${API_FOLDER}/dist
-
-pushd ${API_FOLDER}
-
-# Build and push Docker images
-docker buildx build \
-  --platform linux/amd64 \
-  --tag "$ECR_REPO_URI:latest" \
-  --tag "$ECR_REPO_URI:$GITHUB_SHA" \
-  --push \
-  .
-
-popd
-
-# Build and push Docker images
+# Define extra dependencies needed for API
+API_EXTRA_DEPS="packages/api-sdk/package.json packages/api-sdk/dist \
+  packages/commonwell-sdk/package.json packages/commonwell-sdk/dist \
+  packages/carequality-sdk/package.json packages/carequality-sdk/dist \
+  packages/ihe-gateway-sdk/package.json packages/ihe-gateway-sdk/dist"
```

Comment:
nit: can we keep one entry per line, like it was before, please? It's way easier to read/edit/review. :)